docs/cookbook/executing_long_running_console_commands.md

```diff
@@ -1,57 +1,60 @@
 # Executing long-running console commands
 
-## Description
+This page describes how to execute long-running console commands, to make sure they don't run out of memory.
+Two examples of such commands are a custom import command or the indexing command provided by the [Solr Bundle](../guide/search.md#solr-bundle).
```

Comment:
for:  a custom import command or the indexing command 
please change to: a custom import command and the indexing command